task/testmap.py

```diff
@@ -33,6 +33,7 @@
             'ubuntu-stable',
             'fedora-32',
             'fedora-33',
+            'fedora-34',
```

Comment:
This needs to be a separate PR, and please start with adding it to `_manual`. Like this it would immediately run for all cockpit PRs, but we need to sort out the tests first.